docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
+  
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
+    <AssemblyOriginatorKeyFile>$(ToolsDir)Test.snk</AssemblyOriginatorKeyFile>
```

Comment:
By default the assemblies are signed using Open.snk. However, if we want to register an assembly in the component catalog using `mlContext.ComponentCatalog.RegisterAssembly()`, the assembly needs to pass the following condition:
https://github.com/dotnet/machinelearning/blob/b861b5d64841cbe0f2c866ee7586872aac450a51/src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs#L1025-L1034

Reply:
After discussing with Eric, it could be even simpler not to sign this assembly. We don't need strong naming for the Samples assembly as it should not be referenced by external code anyways.